tests/scripts/optdata/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+"""
+    This script prepares the local source tree to be built with
+    custom optdata. Simply run this script and follow the
+    instructions to inject manually created optdata into the build.
+"""
+
+import argparse
+import json
+import os
+from os import path
+import shutil
+import subprocess
+import sys
+
+# Display the docstring if the user passes -h|--help
+argparse.ArgumentParser(description=__doc__).parse_args()
+
+SCRIPT_ROOT = path.dirname(path.realpath(__file__))
+REPO_ROOT = path.realpath(path.join(SCRIPT_ROOT, '..', '..', '..'))
```

Comment:
I'd be more comfortable with the user being required to pass the repo_root, on the off chance that this file gets moved, or something about the repo structure changes. I doubt it would, but it's something to guard against.